docs/src/components/ImageContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { css } from "styled-components";
+import { defaultTheme } from "@kiwicom/orbit-components";
+
+interface Props {
+  align?: "left" | "center" | "right";
+  border?: boolean;
+  children: React.ReactNode;
+  size?: "small" | "medium" | "large";
+}
+
+const ImageContainer = ({ align = "center", border, size = "large", children }: Props) => {
+  const getContainerWidth = containerSize => {
+    if (containerSize === "small") return "200px";
+    if (containerSize === "medium") return "400px";
+    return "650px";
+  };
+  const containerWidth = getContainerWidth(size);
+
+  const getContainerMargin = containerAlignment => {
+    if (containerAlignment === "right") return "0 0 0 auto";
+    if (containerAlignment === "left") return "0";
+    return "0 auto";
+  };
+  const containerMargin = getContainerMargin(align);
+  return (
+    <div
+      css={css`
+        margin: ${containerMargin};
+        width: ${containerWidth};
```

Comment:
No need to store the return value to a variable:
```suggestion
        margin: ${getContainerMargin()};
        width: ${getContainerWidth()};
```